data/fields/shooting.json

```diff
@@ -1,12 +0,0 @@
-{
-    "key": "shooting",
-    "type": "semiCombo",
-    "label": "Type",
-    "strings": {
-        "options": {
-            "rifle": "Rifle",
-            "clay_pigeon": "Clay Pigeon",
-            "pistol": "Pistol"
```

Comment:
I removed the Type field based on feedback from @westnordost in OSMUS Slack. I was under the impression that the `shooting=*` key indicates a discipline (e.g., [clay pigeon shooting](https://en.wikipedia.org/wiki/Clay_pigeon_shooting)), not a specific type of target or weapon used in the discipline, but I can see how these values are confusing without additional explanation. I removed the field entirely instead of turning it into a freeform combo box, because the most common value, `range`, is considered superfluous.

For what it’s worth, my only personal reference point for this sport is [this shooting club](https://www.openstreetmap.org/way/717832152) that I mapped once. I used `shooting=*` to clarify which range was for which discipline, but the particulars aren’t that important to me. The most important thing is not allowing the School preset to come up first in a search for the terms being added here.